specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -126,6 +126,147 @@
+      "post": {
+        "description": "The operation to create a new WebDirect or EA Azure subscription (under billingProfiles).",
+        "operationId": "Subscriptions_CreateSubscriptionUnderBillingProfile",
+        "x-ms-examples": {
+          "createSubscription": {
+            "$ref": "./examples/createSubscriptionUnderBillingProfile.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionCreationResult"
+            }
+          },
+          "202": {
+            "description": "Accepted. Subscription creation is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "billingAccountName",
+            "in": "path",
+            "description": "The name of the Microsoft Customer Agreement billing account for which you want to create the subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "billingProfileName",
+            "in": "path",
+            "description": "The name of the billing profile in the billing account for which you want to create the subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "invoiceSectionName",
+            "in": "path",
+            "description": "The name of the invoice section in the billing account for which you want to create the subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "The subscription creation parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ModernSubscriptionCreationParameters"
```

Comment:
The type 'ModernSubscriptionCreationParameters' has a required property of 'billingProfileId'. Is this expected to be the same as the parent scope of the action? If so can we compute it (substring of request path) instead of requiring the user to declare it in the body as well?

Reply:
Yes, both are same. As of today, reusing the same input object for existing API and the new one.

Reply:
And to update: we are making change to compute the billingProfile from request URI in our GA release with 2018-11-01 api-version.